app/src/main/java/uk/ac/ebi/atlas/experimentimport/admin/SingleCellExperimentAdminController.java

```diff
@@ -27,4 +35,13 @@ public SingleCellExperimentAdminController(DataFileHub dataFileHub,
     public String fooBar() {
         return "";
     }
+
+    @GetMapping(value = "/{experimentAccession}")
+    public void updateSingleCellExperiment(
+            @PathVariable String experimentAccession,
+            @RequestParam(value = "isPrivate") Boolean isPrivate,
+            ScxaExperimentCrud scxaExperimentCrud) {
+        UUID accessKeyUUID = scxaExperimentCrud.createExperiment(experimentAccession, isPrivate);
+        LOGGER.info("Import experiment successfully, accessKeyUUID: {}", accessKeyUUID.toString());
+    }
```

Comment:
Please update this logger message accordingly, does it create a new experiment or update experiment & clear cache?

Reply:
We import an experiment. If this experiment is existed in the cache, we'll update it, if not, we'll add it. So `import` will be a proper logger message and I take the reference from where we call `createExperiment` in the existing code.

Reply:
Please rename controller method name as a `createOrUpdateExperiment()` because `createExperiment()` from crud is doing create/update.

We can remove the logger in the controller if it passes the same info from the DAO class.

If you want to keep it -  update it like below 

```
LOGGER.info("Imported/updated experiment successfully, accessKeyUUID: {}", accessKeyUUID.toString());

```
All good.